topics/transcriptomics/tutorials/clipseq/tutorial.md

```diff
@@ -0,0 +1,629 @@
+>
+>    > ### {% icon comment %} MEME and DREME
+>    >
+>    > **MEME** and **DREME** are two motif finding tools that MEME-ChIP uses. More information about theses tools can be found [here](http://meme-suite.org/).
+>    {: .comment}
+>
+{: .hands_on}
+>
+> ### {% icon question %} Questions
+>
+> 1. What is the purpose of SlopBED?
+> 2. What is the meaning of the E-value?
+> 3. What does the sequence plots in the **MEME-ChIP** html mean?
+> 4. Click on the link under the column **Discovery/​Enrichment Program** for two of the motifs one from **MEME** and one from **DREME**. What is the site-count for the most significant motif? How many sequences have we checked for **MEME** and for **DREME**? What is the fraction of sequences that covered that motif? (MEME-ChIP version 4.11.2)
+> 5. Have we found the motif `TGCATG`?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. **PEAKachu** might underestimate the length of the binding regions, because the actually binding concentrates on the crosslinking site that is one nucleotide long. Thus, we extend the peaks from **PEAKachu** by 20 bp at each site. A true conserved binding motif will not be affected by it, if we make the region to wide.
+> > 2. The E-value represents the expected number of times we would find our sequence motif in a database (peak file), just by chance. This means, that a small E-value correspond to a very significant motif, because the expected number we would find that motif in our peakfile just by chance is very low. Other sequences like repeats will have a high E-value on the other hand.  
+> > 3. The x-axis of the sequence plots represents the nucleotide position of the motif. The y-axis stands for the total information (uncertainty) of each position and thus stands for the probability that the nucleotide at a certain position is the specific letter (for DNA: T,C,G,A). Bigger letters stand for a higher probability. For more information read [here](https://en.wikipedia.org/wiki/Sequence_logo).
+> > 4. For the result with MEME-ChIP version 4.11.2: **MEME** site count = 37, total sequences = 100, fraction = 37%; **DREME** site count = 103, total sequences = 247, fraction = 41.7%.
+> > 5. Yes.
+> {: .solution}
+>
+{: .question}
+
+## Functional analysis with **RNA Centric Annotation System**
+
+> ### {% icon hands_on %} Hands-on: Functional analysis
+>
+> 1. **RNA Centric Annotation System (v. 1.1.1)** {% icon tool %} with the following parameters:
+>    - *"Genome Version"*: `hg19`
+>    - {% icon param-file %} *"Target regions in BED format"*: `bed` (output of **SlopBed** {% icon tool %})
+>    - {% icon param-file %} *"Reference annotation in ENSEMBL GTF format"*: `Homo_sapiens.GRCh37.74.gtf` (file from your history)
+>    - *"Run annotation."*: `Yes`
+>    - *"Run GO term enrichment"*: `Yes`
+>    - *"Run gene set enrichment"*: `No`
+>    - *"Run motif search"*: `No`
+>    - *"Downsampling (N)"*: `"0"`
+> 2. View the output html of **RNA Centric Annotation System**
```

Comment:
Can you include the output, odes it make sense. If not please explain the output.